### PR TITLE
kubeadm: Hide the unnecessary --fuzz-iters flag

### PR DESCRIPTION
**Pull request by luxas**
super straightforward. We don't want this flag to leak into our UX.

cc @jbeda @dmmcquay @deads2k

### CODE REVIEW

File: cmd/kubeadm/app/kubeadm.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ func Run() error {
 	// We do not want these flags to show up in --help
 	pflag.CommandLine.MarkHidden("google-json-key")
 	pflag.CommandLine.MarkHidden("log-flush-frequency")
+	pflag.CommandLine.MarkHidden("fuzz-iters")
```
Comment (liggitt) [inline cmd/kubeadm/app/kubeadm.go]: if this is showing up, doesn't it mean kubeadm is linking to the api/testing package somehow? seems like the right fix would be to break that import chain

Comment (luxas) [inline cmd/kubeadm/app/kubeadm.go]: Obviously, yes, was too tired yesterday when uploading this I didn't even think about it :)
Thanks for forcing me to check the import chain, now fixed

**Comment by luxas** [comment]
This is a little bit irritating; @deads2k should obviously be authorized to approve pkg/api/testing, but seems that file hasn't got any approvers, just reviewers.
Anyway, applied the label manually

**Comment by liggitt** [comment]
API sub packages intentionally are lacking approvers to ensure api approval hits the @kubernetes/api-approvers group. The testing subpackage should probably be an exception.

**Pull request merged**